src/components/AudioPlayer/HighlightedLocationUpdater.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useRef } from 'react';
```

Comment:
This component can probably just become a hook, since it doesn't really do any rendering